iidm/iidm-reducer/src/main/java/com/powsybl/iidm/reducer/DefaultNetworkReducerBuilder.java

```diff
@@ -0,0 +1,55 @@
+/**
+ * Copyright (c) 2019, RTE (http://www.rte-france.com)
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/.
+ */
+package com.powsybl.iidm.reducer;
+
+import java.util.ArrayList;
+import java.util.Collection;
+import java.util.List;
+import java.util.Objects;
+
+/**
+ * @author Mathieu Bague <mathieu.bague at rte-france.com>
+ */
+public class DefaultNetworkReducerBuilder {
+
+    private NetworkFilter filter;
+
+    private ReductionOptions options = new ReductionOptions();
+
+    private List<NetworkReducerObserver> observers = new ArrayList<>();
+
+    public DefaultNetworkReducerBuilder setNetworkFilter(NetworkFilter filter) {
```

Comment:
'with' syntaxe is more common with builders. I personnaly find it more fluent.

Reply:
Done